internal/apiserver/pgdumpservice/pgdumpimpl.go

```diff
@@ -247,8 +247,10 @@ func ShowpgDump(clusterName string, selector string, ns string) msgs.ShowBackupR
 }
 
 // builds out a pgTask structure that can be handed to kube
-func buildPgTaskForDump(clusterName string, taskName string, action string, podName string,
-	containerName string, request *msgs.CreatepgDumpBackupRequest) *crv1.Pgtask {
+func buildPgTaskForDump(clusterName, taskName, action, podName, containerName string,
+	request *msgs.CreatepgDumpBackupRequest) *crv1.Pgtask {
+
+	log.Debugf("REQUEST VALUES: %v", request)
```

Comment:
Is there anything sensitive that could be contained in here (e.g. passwords?)

Reply:
I'll remove this in the final PR. I'm still testing this as a 'draft'. Sorry for any confusion here.